org.bndtools.launch/src/bndtools/launch/AbstractOSGiLaunchDelegate.java

```diff
@@ -294,6 +296,13 @@ public String[][] getClasspathAndModulepath(ILaunchConfiguration config) throws
 			classpathAndModulepath = new String[2][];
 			classpathAndModulepath[1] = new String[0];
 		}
+		if (classpathAndModulepath[1].length > 0) {
```

Comment:
```suggestion
		} else if (classpathAndModulepath[1].length > 0) {
```

This test is only interesting on the else side of the if.